lib/matplotlib/pyplot.py

```diff
@@ -1406,6 +1406,12 @@ def axis(*v, **kwargs):
     as kwargs selectively to alter just those limits without changing
     the others.
 
+      >>> axis('square')
+
+    changes the limit ranges (*xmax*-*xmin*) and (*ymax*-*ymin*) of
+    the *x* and *y* axes to be the same, and have the same scaling,
+    resulting in a square plot.
+
```

Comment:
You are two for two in confusing devs on this!

Which does raise the question why isn't this part of boilerplate generated
code?

On Mon, Apr 20, 2015, 18:40 basharovV <notifications@github.com> wrote:

> In lib/matplotlib/pyplot.py
> <https://github.com/matplotlib/matplotlib/pull/4310#discussion_r28736156>:
>
> > @@ -1406,6 +1406,12 @@ def axis(*v, **kwargs):
> >      as kwargs selectively to alter just those limits without changing
> >      the others.
> >
> > +      >>> axis('square')
> > +
> > +    changes the limit ranges (*xmax*-*xmin*) and (*ymax*-*ymin*) of
> > +    the *x* and *y* axes to be the same, and have the same scaling,
> > +    resulting in a square plot.
> > +
>
> It's outside the autogenerated part, running boilerplate.py doesn't affect
> it.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/matplotlib/matplotlib/pull/4310/files#r28736156>.
>